src/PhoneNumber/PhoneNumberTypeEnum.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace DR\Internationalization\PhoneNumber;
+
+enum PhoneNumberTypeEnum: string
+{
+    case FIXED_LINE = 'FIXED_LINE';
```

Comment:
seeing you need to convert twice to go from `numberType` to enum. change this enum to:

`case FIXED_LINE = PhoneNumberType::FIXED_LINE`

and change enum to `PhoneNumberTypeEnum: int`

Reply:
As discussed, not possible because some php limits.